fn/testfn.go

```diff
@@ -134,7 +134,7 @@ func runlocaltest(target string, in, expectedOut, expectedErr *string, env map[s
 		restrictedEnv = append(restrictedEnv, k)
 	}
 
-	if err := runff(target, stdin, &stdout, &stderr, restrictedEnv); err != nil {
+	if err := runff(target, stdin, &stdout, &stderr, restrictedEnv, nil); err != nil {
```

Comment:
Note that I didn't add test support, though it might be pretty trivial from what I saw. I'm not sure if that alone would be sufficient to allow example/postgres/test.sh to be a 'fn test' configuration instead (since those exist in the func.yaml, which isn't checked in), but there may be other compelling reasons.